README.md

```diff
@@ -150,6 +150,10 @@ Run `docker-compose up -d` instead.
 
 * Permission problem? See [this doc (Setting up Permission)](http://symfony.com/doc/current/book/installation.html#checking-symfony-application-configuration-and-setup)
 
+* How to config Xdebug?
+Xdebug is configured out of the box!
+Just config your IDE to connect port  `9001` and id key `PHPSTORM`
```

Comment:
Done that. Configured a server with path mappings etc.
Don't know what I'm missing. I'd love to give maxpou the feedback that this works so he can accept the PR, but I'm too dumb to get it working. :D